### ISSUE TITLE
Docs: Clean up installation chapter

### ISSUE THREAD

**Issue by caponica** [sonata-project/SonataAdminBundle#1520]
Improve [this chapter](https://github.com/sonata-project/SonataAdminBundle/blob/master/Resources/doc/reference/installation.rst) and make it more accessible to newcomers by making the following changes:
- clearly show entries that could be added to an existing project's composer.json (and other config files) that would result in a complete working installation of SonataAdmin and all _required_ dependencies, including an example persistency service (e.g. DoctrineORM)
- explain how to vary that example to get the other persistency services working
- document optional add-ons, what they do and what should be added to the composer and config files to add them in.

In short, create **step by step instructions** that a newbie can follow. Refer to other bundles' docs, and tell the reader to go there for the latest information, but put the basics here in a complete, working, configuration.

At the moment, given the state of SonataAdmin, it may be that some of those dependencies are included as "dev-master" but it would be best to go through and refactor this example once a stable release is made that works with Symfony-2.3.x.

**Comment by jordisala1991** [comment]
I think this can be closed. The install chapter is already good enough.

It explains how to install admin and related bundles.
It show the persistence layers and points out to their docs.
It shows required config and routes.

https://sonata-project.org/bundles/admin/3-x/doc/getting_started/installation.html

You can check it out.

If anyone think it needs improvement. Please open a new issue with the exact missing information.

Closing
